build.sh

```diff
@@ -8,4 +8,14 @@
 # SPDX-License-Identifier: EPL-2.0
 #
 
-docker build -t quay.io/eclipse/che-devfile-registry:nightly .
+VERSION=$(head -n 1 VERSION)
+case $VERSION in
+  *SNAPSHOT)
+    echo "Building plugin registry"
+    # docker build -t quay.io/eclipse/che-devfile-registry:nightly .
+    ;;
+  *)
+    echo "Building plugin registry for release ${VERSION}"
```

Comment:
maybe add info that for the release version  `RELEASE` version of plugin runtime containers will be used?